### PR TITLE
Display user county cog 87

### PR DESCRIPTION
**Pull request by kelfink**

### CODE REVIEW

**Comment by kelfink** [comment]
retest this please

**Comment by kelfink** [comment]
retest this please

File: app/javascript/containers/usersListContainer.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
 import { connect } from 'react-redux';
 import UsersList from '../views/usersList/UsersList.jsx';
-import * as actionTypes from '../actions/actionTypes';
-import { selectUserRecords } from '../selectors/userListSelector';
+import { selectUserRecords, selectCounty } from '../selectors/userListSelector';
 
 const mapStateToProps = state => {
   return {
     userList: selectUserRecords(state),
+    account_county: selectCounty(state),
```
[the remark this answers is not in the document]
Comment (Sushmareddy-OSI) [reply app/javascript/containers/usersListContainer.js]: +1

[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/containers/usersListContainer.js]: whoops good catch!

**Comment by kelfink** [comment]
retest this please

**Pull request merged**